packages/SwingSet/test/test-device-bridge.js

```diff
@@ -24,12 +23,13 @@ test('bridge device', async t => {
     bootstrap: 'bootstrap',
     vats: {
       bootstrap: {
-        sourceSpec: require.resolve('./device-bridge-bootstrap.js'),
+        sourceSpec: new URL('device-bridge-bootstrap.js', import.meta.url)
```

Comment:
You will note here and elsewhere that the `./` prefix is only relevant in the domain of module specifiers so I’ve removed it when the domain is URL’s.